addons/hr_expense/static/src/views/list.js

```diff
@@ -69,7 +74,13 @@ export class ExpenseListController extends ListController {
         await this.model.root.load();
         this.render(true);
     }
-
+    async createReport () {
+        const records = this.model.root.selection;
+        const res = await this.orm.call(this.model.rootParams.resModel, 'get_expenses_to_submit', [records.map((record) => record.resId)]);
```

Comment:
by consistency with the rest of odoo, such action should be named `action_show_expenses_to_submit`